config/prometheus/monitor.yaml

```diff
@@ -12,7 +12,6 @@ spec:
     - path: /metrics
       port: https
       scheme: https
```

Comment:
What is missing for having a TLS without skipping verify?
It's basically the same reason why we have this bug: https://issues.redhat.com/browse/NETOBSERV-765

Reply:
Similar to what we did for FLP we need to add the labels to the metric services to generate certificate and mount them in the kube-rbac-proxy container so they can be used.